### ISSUE TITLE
Option to blacklist other types of projectiles (ex; loader's grappling fists)

### ISSUE THREAD

**Issue by sempervali** [Goorakh/RoR2Mods#7]
As the title says. Would it be possible to have some system for manually blacklisting specific projectiles from being randomized? 

Being able to disable just Loader's fists from being randomized so she can still use her primary ability would be the primary use I see something like this being used for.

**Comment by LockeColeFF** [comment]
Yeah, the projectile randomizer is the most fun part of this mod but it has a tendency to break the game.  There are literally hundreds of types of projectiles so a blacklist of each one might not be the best idea.  Though in the case of the loader it makes sense.

Possible workaround for the game crashing due to projectiles would be to make it player-side only.  Having to randomize a handful of projectiles for the player and the summons is way less likely to crash the game than randomizing EVERY projectile in the game.

**Comment by Goorakh** [comment]
I did want to add this when I made the "blacklist update", I ended up not doing it mostly since there is no maintained list of projectile names or ID's anywhere I'm aware of, and if there wasn't a way to find out what a specific projectle's internal name is the setting would be pretty useless.

I would like to say I'll get a list like that together eventually, but as of right now I've kinda lost motivation for working on this mod, and I consider it "done" for now, every major feature I wanted to add has been added, and most if not all of the major bugs have been ironed out.

The mod also being the very first RoR2 mod I've made means it's not lacking in some questionable code simply due to me not being too familiar with modding at the time, which makes maintaining it tedious at best.

This is already on my list of features though, so if I get the urge to work on the mod again it'll probably be one of the first things I add. Can't promise I will though, so don't get your hopes up too much, sorry :/

That being said... PR's are open if you're able to contribute :)
